build.gradle

```diff
@@ -4,19 +4,22 @@ plugins {
 
 repositories {
     mavenCentral()
+    flatDir dirs: "${rootDir}/lib"
 }
 
 dependencies {
-    implementation 'junit:junit:4.13.2'
+    implementation 'junit:junit:5.9.0'
```

Comment:
The autograder doesn't like this dependency change. It keeps saying "Could not find junit:junit:5.9.0". We should switch it back to 4.13.2.